themes/theme-default/src/client/styles/code.scss

```diff
@@ -229,7 +229,6 @@ div[class*='language-'] {
       padding-top: 1.3rem;
       font-size: 0.875em;
       line-height: $line-height;
-      counter-reset: line-number;
```

Comment:
239-242 should be removed?

Reply:
Do not remove, currently using `counter-increment` for line numbering.

Reply:
I  think its ok for you to use actual numbers like vitepress.

Reply:
When writing numbers into the DOM, `plugin-seo` includes the numbers from `line-numbers` in the `og:description`, but it does not include the `code blocks content`, which seems a bit strange. Perhaps `plugin-seo` needs to make some adjustments.